core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/date/GetYesterdayDateUnitTest.java

```diff
@@ -0,0 +1,89 @@
+package com.baeldung.date;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.joda.time.Instant;
+import org.junit.jupiter.api.Test;
+
+class GetYesterdayDateUnitTest {
+
+    @Test
+    void givenDate_whenUsingCalendarClass_thenReturnYesterday() {
```

Comment:
I would keep the order of test methods in alignment with the article's sections

Reply:
Done